internal_task_handlers.go

```diff
@@ -892,7 +892,7 @@ func (i *cadenceInvoker) Heartbeat(details []byte) error {
 
 func (i *cadenceInvoker) internalHeartBeat(details []byte) (bool, error) {
```

Comment:
I thought the idea is for user to pass in the context.  Don't you need to update internalHeartBeat to also take in context along with details?

Reply:
no, this change only affect our Client/DomainClient interface. Poll task and respond task completed are not exposed through our Client interface. Workers are long running process and does not take context from user.